pubspec.yaml

```diff
@@ -1,6 +1,7 @@
 name: flutter_billing
 description: A flutter plugin to communicate with billing on iOS and Android.
-version: 0.2.3
+version: 0.2.4
```

Comment:
There is release.md once merged in I will do release update. Please exclude this change from this commit.